api/pom.xml

```diff
@@ -23,22 +23,27 @@
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.5</version>
+        <relativePath/>
     </parent>
 
     <groupId>jakarta.servlet</groupId>
     <artifactId>jakarta.servlet-api</artifactId>
-    <version>4.0.2-SNAPSHOT</version>
+    <version>4.0.3-SNAPSHOT</version>
```

Comment:
Version number increase should have been done as part of the release process. Bad form to mix it in with substantive changes

Reply:
I was doubting whether this should be done here, I'll revert it then.

Reply:
Okay, reverted.